release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2913,3 +2913,36 @@ $ oc adm release info 4.11.9 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-12"]
+=== RHSA-2022:7201 - {product-title} 4.11.12 bug fix and security update
+
+Issued: 2022-10-31
+
+{product-title} release 4.11.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:7201[RHSA-2022:7201] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7200[RHBA-2022:7200] advisory.
```

Comment:
```suggestion
{product-title} release 4.11.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:7201[RHSA-2022:7201] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7200[RHBA-2022:7200] advisory.
```

Reply:
I don't disagree wit this suggestion. Unfortunately, this is stock text that I can't change unless I change it across all branches' RNs.

Reply:
Gotcha! No worries, yeah that would be too much for this small nit! 👍